src/main/resources/dockerfiles/Dockerfile-skeleton

```diff
@@ -90,4 +91,6 @@ WORKDIR /opt/alfresco
 
 HEALTHCHECK --interval=10s --timeout=3s --retries=30 --start-period=20s CMD curl -f http://localhost:${TOMCAT_PORT}/alfresco/s/api/server || exit 1
 
+LABEL xenitVersion=$XENIT_VERSION
```

Comment:
Can you make this label something like `eu.xenit.docker.docker-alfresco.version`? Docker labels are supposed to be namespaced.

Reply:
You are a maintainer as well, it's faster to just do it than asking me?